website/docs/reference/global-configs/warnings.md

```diff
@@ -2,129 +2,216 @@
 title: "Warnings"
 id: "warnings"
 sidebar: "Warnings"
+toc_max_heading_level: 2
+intro_text: "dbt's warnings can be converted to errors using the --warn-error flag to promote all warnings to errors or --warn-error-options for granular control through options."
```

Comment:
can convert warnings to error or silence them - so it's more about configuring the severity (?) of these warnings (i'm not totally sure the right way to phrase that)

Reply:
What about:

Use the --warn-error flag to promote all warnings to errors or --warn-error-options for granular control through options.